neon_llm_chatgpt/chatgpt.py

```diff
@@ -72,20 +84,20 @@ def get_sorted_answer_indexes(self, question: str, answers: List[str]) -> List[i
         """
         if not answers:
             return []
-        scores = self._score(question=question, answers=answers)
+        scores = self._score(prompt=question, targets=answers)
```

Comment:
Is this a change in the `NeonLLM` base class and is there a new minimum version requirement that goes with this?

Reply:
Currently we set `neon_llm_core==0.0.6` as format may change